src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,32 @@
-export const PostInfo = () => <>Put the post here</>;
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/extensions */
+/* eslint-disable indent */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable prettier/prettier */
+/* eslint-disable react/jsx-no-useless-fragment */
```

Comment:
Consider removing unnecessary ESLint disable comments if they are not needed. This will help keep the code clean and maintainable.